superset/utils/pandas_postprocessing/contribution.py

```diff
@@ -71,5 +73,7 @@ def contribution(
     numeric_df = numeric_df[columns]
     axis = 0 if orientation == PostProcessingContributionOrientation.COLUMN else 1
     numeric_df = numeric_df / numeric_df.values.sum(axis=axis, keepdims=True)
+    # replace infinity and nan with 0 in dataframe
+    numeric_df.replace(to_replace=[np.Inf, -np.Inf, np.nan], value=0, inplace=True)
```

Comment:
Are we sure we want to display infinities as 0? I think having `1/0` to result in `null` to be more intuitive. I assume keeping infinities as null would cause a break in the chart as opposed to it dropping to zero, which IMO feels more intuitive.

Reply:
let us discuss this in new PR.